lib/Braintree/OAuthGateway.php

```diff
@@ -70,7 +71,8 @@ public function _mapError($result)
         } else if ($error->code == Braintree_Error_Codes::OAUTH_INVALID_SCOPE) {
             $result->error = 'invalid_scope';
         }
-        $result->errorDescription = explode(': ', $error->message)[1];
+        $descriptions = explode(': ', $error->message);
+        $result->errorDescription = $descriptions[1];
```

Comment:
The original code from Braintree does not make this check so I'm inclined to leave it out.